discovery-infra/start_discovery.py

```diff
@@ -469,6 +440,9 @@ def main():
         type=str,
         default='assisted-service'
     )
+    parser.add_argument(
+        "--skip-vip-dhcp-allocation", help="Disable VIP DHCP allocation mode", action="store_true"
```

Comment:
what will happened if we skip dhcp allocation?

Reply:
Same old VIPs allocation